pytorch_blade/torch_blade/config.py

```diff
@@ -177,6 +177,7 @@ def __init__(self):
         self._customize_op_black_list = []
         self._customize_jit_passes = []
         self._opt_pipeline = 'DISC'
+        self._enable_tf32 = False
```

Comment:
Shall we enable tf32 by default to align with Pytorch (before 1.12)?

Reply:
+1

Reply:
Done.

Reply:
But at least we should have a toggle to disable it when precision problem occurs? I saw it's hard-coded.